front/app/services/projectModerators.ts

```diff
@@ -24,7 +24,10 @@ export async function deleteProjectModerator(
   );
   invalidateSeatsCache();
   await streams.fetchAllWith({
-    apiEndpoint: [`${API_PATH}/projects/${projectId}/moderators`],
+    apiEndpoint: [
+      `${API_PATH}/projects/${projectId}/moderators`,
+      `${API_PATH}/users`,
```

Comment:
Works without refetching `${API_PATH}/users` + we don't have this for folder mods, so perhaps delete it? It also keeps project and folder mod services aligned.

Reply:
I've already pushed this to save you time. If I'm mistaken, please correct me again.

Reply:
We need that line because without it, deleting a moderator means that moderator won't show up in the users list

Reply:
True! So we also need it for folder mods then.

Reply:
I'll add it back.

Reply:
No, It works well for folders. No need to make the change there

> So we also need it for folder mods then.

Reply:
I've just checked and this works fine for both project and folder moderators without refetching users.

Reply:
So good to merge then?

Reply:
It doesn't work for me on project without refetching users. Not sure what the issue there might be

Reply:
This is small and costing us time though. So we can merge it and it to the  feature test ticket